supermarq/features.py

```diff
@@ -0,0 +1,167 @@
+from typing import Union
+
+import cirq
+import networkx as nx
+import numpy as np
+import qiskit
+
+
+def cirq_to_qiskit(circuit: cirq.Circuit) -> qiskit.circuit.QuantumCircuit:
```

Comment:
We might want to remove this and use SuperstaQ's `cirq_to_qiskit` which is more robust. This will require a token to use though. Maybe we want to keep it for people who don't have a token?

Reply:
@vtomole this is a dumb question but: in order for a user to use `superstaq.cirq_to_qiskit` they would need to import SuperstaQ, but it's a private repo?

How can we give external users access to this function?

Reply:
We'll add a new [API endpoint](https://superstaq.super.tech/api). A good reference is to rip off one of the `_compile` ones and call off to  `superstaq.cirq_to_qiskit` under the hood.

Reply:
Does this hacky `cirq_to_qiskit` work for everything here now? I'm starting to lean that we let it be for agility's sake and address the endpoints once we have the tutorial set up. The endpoints are a bonus.

Reply:
It works for the current SupermarQ benchmarks. This is the same file I used to generate the feature plots shown in the paper.

Reply:
Okay. Let's get this in and prioritize putting together the tutorial for now. Let me review other parts of this PR.

Reply:
Sounds good, I will actually move `cirq_to_qiskit` out into its own file: `converters.cirq_to_qiskit` because we will also want to add the option to convert circuits when the benchmarks themselves are created.

Reply:
Let's get the checks to pass as well.

Reply:
> Currently, the radar_factory function within feature_plotting.py is too complex - not sure what to change to accommodate this.

Could you take a look at this